<launch>
  <group ns="FleaCam">
    <group ns="left">
      <node name="left_dev" pkg="camera1394" type="camera1394_node"
	    respawn="false" output="screen"
	    clear_params="true">
	<param name="video_mode" value="640x480_rgb8"/>
	<param name="fps" value="15"/>
	<param name="guid" value="00b09d01008c8f46"/>
	<param name="frame_id" type="string" value="/left_flea_cam"/>
	<param name="camera_info_url"
	       value="file:///$(find cr_capture)/launch/2010-02-16-15-14-42-stereo-left.yaml"/>
      </node>
      <node name="image_proc" pkg="image_proc" type="image_proc"
	    ns="camera"/>
      <!--
      <node name="iview" pkg="image_view" type="image_view">
	<remap from="image" to="camera/image_rect_color"/>
      </node>
      -->
    </group>

    <group ns="right">
      <node name="right_dev" pkg="camera1394" type="camera1394_node"
	    respawn="false" output="screen">
	<param name="fps" value="15"/>
	<param name="video_mode" value="640x480_rgb8"/>
	<param name="guid" value="00b09d01008c8f44"/>
	<param name="frame_id" type="string" value="/right_flea_cam"/>
	<param name="encoding" value="rgb8"/>
	<param name="camera_info_url"
	       value="file:///$(find cr_capture)/launch/2010-02-16-15-14-42-stereo-right.yaml"/>
      </node>
      <node name="image_proc" pkg="image_proc" type="image_proc"
	    ns="camera"/>
      <!--
      <node name="iview" pkg="image_view" type="image_view">
	<remap from="image" to="camera/image_rect_color"/>
      </node>
      -->
    </group>
  </group>

</launch>
